_data/speakers.yml

```diff
@@ -109,5 +131,41 @@
     - {name: "twitter", link: "https://twitter.com/dorait"}
     - {name: "site", link: "http://littlebitsofknowledge.com"}
 
+-
+  id: 9
```

Comment:
Can we have all these probable speakers details commented so that they don't show as blank boxes ?